fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -701,6 +701,13 @@ class ShardManager {
 					if (!shard->initialized()) {
 						continue;
 					}
+					uint64_t liveDataSize = 0;
+					ASSERT(shard->db->GetIntProperty(
+					    shard->cf, rocksdb::DB::Properties::kEstimateLiveDataSize, &liveDataSize));
+					TraceEvent(SevInfo, "PhysicalShardCFSize")
+					    .detail("ShardId", id)
```

Comment:
Do we have other logs that will connect this physical shard to a storage server and/or key ranges?